.gitignore

```diff
@@ -2,3 +2,4 @@ _bookdown_files
 WSBIM2122.Rmd
 shell_data
 wsbim2122_data
+docs/
```

Comment:
Definitely not! The `docs/` directory must be on github, given that that's the one than holds the html version of the course.

Reply:
I'll fix it.